cmake/CMakeLists.txt

```diff
@@ -135,11 +148,11 @@ if(onnxruntime_ENABLE_LTO)
       message(WARNING "IPO is not supported by this compiler")
       set(onnxruntime_ENABLE_LTO OFF)
     else()
+      set (CMAKE_INTERPROCEDURAL_OPTIMIZATION ON)
       if(UNIX)
         set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -flto")
         set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -flto")
         set (CMAKE_LDFLAGS "${CMAKE_CXX_FLAGS} -flto")
-        set (CMAKE_INTERPROCEDURAL_OPTIMIZATION ON)
```

Comment:
Why do you need both `CMAKE_INTERPROCEDURAL_OPTIMIZATION` and setting `-flto` manually?

Reply:
Good Question!

Reply:
I'll remove it.